src/ocean/Compute.ts

```diff
@@ -244,39 +243,60 @@ export class Compute extends Instantiable {
       did,
       consumerAccount,
       jobId,
-      txId,
-      sign
+      txId
     )
 
     return computeJobsList as ComputeJob[]
   }
 
   /**
-   * Returns the final result of a specific compute job published as an asset.
+   * Downloads compute job result
    * @param  {Account} consumerAccount The account of the consumer ordering the service.
-   * @param  {DDO|string} asset DID Descriptor Object containing all the data related to an asset or a Decentralized identifier.
-   * @param  {string} jobId The ID of the compute job to be stopped.
-   * @return {Promise<ComputeJob>} Returns the DDO of the result asset.
+   * @param  {string} jobId JobId
+   * @param  {number} index Compute Result index
+   * @param  {string} destination: destination folder
```

Comment:
What destination folder is meant here? Where is this folder created? We should have a default here, or have this not as a param at all.

Reply:
it's for when you use it on a server. This is how it's always been, no need to refactor now just for the sake of refactoring.